cardano_node_tests/utils/dbsync_utils.py

```diff
@@ -883,6 +890,21 @@ def check_tx(
             r.serialised_size > 0 for r in db_plutus_scripts
         ), f"The `serialised_size` <= 0 for some of the Plutus scripts:\n{db_plutus_scripts}"
 
+    db_timelock_scripts = {r for r in response.scripts if r.type == "timelock"}
```

Comment:
Why do we need special case for "timelock" scripts? Ideally we should check whatever scripts were present in the transaction.

Reply:
Makes sense, so I joined the check related to plutus script (lines above) and also added a check for minting scripts, let me know if is out of scope.